wordpress-mobile/WordPress-iOS
# Quick Start: New visuals v2

Part of #18621

## Description
- Updated the Quick Start card design to match the original proposed design
- Design ref: pbArwn-4wS-p2

## Context
- We initially decided to modify the original proposed design to a simpler design (Ref: p1652976178640989/1652793339.295619-slack-C027K4MNPGQ)
- Since there was some time left over in the sprint, I looked into matching the original proposed design (Ref: p1653496943994179/1652793339.295619-slack-C027K4MNPGQ)

## How to test

1. Set the initial screen to "Home" via the App Settings
2. Enable Quick Start for Existing Users via the debug menu
3. ✅ Verify: The Quick Start card design matches the original proposed design
4. Tap the ellipsis button then tap "Remove Next Steps"
5. ✅ Verify: The Quick Start card is removed from the dashboard
6. Set the initial screen to "Menu" via the App Settings
7. Enable Quick Start for Existing Users via the debug menu
8. ✅ Verify: The Quick Start card design matches the original proposed design -- and that the leading / trailing margins of content inside the Quick Start card is the same (i.e. no gap on the trailing edge)
9. Switch to a site that has no posts
10. ✅ Verify: The No Posts card design remains the same as before (i.e. no gap on the trailing edge)

![Simulator Screen Shot - iPhone 13 - 2022-05-26 at 16 27 05](https://user-images.githubusercontent.com/6711616/170523006-7bed0979-1925-4dd6-b114-ac0f2fd8815f.png) | ![Simulator Screen Shot - iPhone 13 - 2022-05-26 at 16 27 25](https://user-images.githubusercontent.com/6711616/170523018-56b56b31-c7c7-457e-8f67-919387c41171.png) | ![Simulator Screen Shot - iPhone 13 - 2022-05-26 at 16 27 38](https://user-images.githubusercontent.com/6711616/170523029-df2c40c7-76c2-403d-8aab-06542c68f571.png) | ![Simulator Screen Shot - iPhone 13 - 2022-05-26 at 16 27 44](https://user-images.githubusercontent.com/6711616/170523033-d93ad215-9a56-494b-baca-59dc7cddde24.png)
-- | -- | -- | --



## Regression Notes
1. Potential unintended areas of impact
- Dashboard cards without headers

9. What I did to test those areas of impact (or what existing automated tests I relied on)
- Tested the steps above

11. What automated tests I added (or what prevented me from doing so)
- n/a

PR submission checklist:
